dataland-frontend/tests/component/components/general/SlideShow.cy.ts

```diff
@@ -0,0 +1,86 @@
+import TestSlideShow from "./TestSlideShow.vue";
```

Comment:
This feels wrong.
Isn't it somehow possible to test the Slideshow by mounting some actual frontend component that uses it?
This feels like production-code, only for a test to work.

Reply:
Well the test slide show contains the actual slide show. I do not see a reason against this. My motivation was to just test the component independent on a specific use case which might change over time and therefore make the test infeasible.

Reply:
Well you have commited a Vue-component into our production code, which is not used / dead for users of our product.
I find this a very hard pill to swallow, since our team rule says, that we don't write stuff into production code, just to please some test (exception: data-test markers).

Since the only purpose of this vue component is to carry the "SlideShow", so that you can mount it in a component test, it should definitely not live in our production code.

My personal alternatives/suggestions on how to avoid the necessity for this file (we can also talk about this in our call):
- Directly mount "SlideShow" and insert some dummy cards into its slot.
- Use one of the two frontend components that use the SlideShow to test it.

Reply:
The component is in the test code but lets discuss this.